tools/overlayhelpers/colliderinit.py

```diff
@@ -397,7 +404,7 @@ def GetColliderInit(address, type, num):
     if fileResult[0] is None:
         return("Invalid address")
 
-    print(hex(fileResult[1]), "in", fileResult[0].split(os.sep)[-1])
+    # print(hex(fileResult[1]), "in", fileResult[0].split(os.sep)[-1])
```

Comment:
Is this line supposed to be commented?

Reply:
Hm. I think I'll delete the previous comment (which is superseded by the error message in `actor_symbol`), but keep this line: it is a useful check that you're in the right file, if nothing else.